lib/Storage/BeeSwarm.php

```diff
@@ -72,7 +72,7 @@ class BeeSwarm extends Common
 	public function __construct($params)
 	{
 		$this->parseParams($params);
-		$this->id = 'ethswarm::' . $this->api_url;
+		$this->id = 'ethswarm::' .str_replace("https://", "", $this->api_url) . '::'.$this->access_key;
```

Comment:
Actually, the api_url should be always pointing to the same location, thus we can get rid of it to shorten these data store in the db and in fact just link them with the token/access_key.

Reply:
Agree! Even if we have a usecase in the future (not sure what it could be right now) that a used needs to point to multiple locations, the token is enought to diferentiate. Also the field id is limited to 64 chars, so I agree on the improvement.